serving-tiles/manually-building-a-tile-server-20-04-lts.md

```diff
@@ -165,7 +165,7 @@ Then we convert the carto project into something that Mapnik can understand:
 You now have a Mapnik XML stylesheet at /home/renderaccount/src/openstreetmap-carto/mapnik.xml .
 # Loading data
 
-Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 17Mb.
+Initially, we'll load only a small amount of test data. Other download locations are available, but "download.geofabrik.de" has a wide range of options. In this example we'll download the data for Azerbaijan, which is about 32Mb.
 
 Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2020-05-02T20:59:02Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```

Comment:
The extract timestamp also needs to be updated
```suggestion
Browse to https://download.geofabrik.de/asia/azerbaijan.html and note the "This file was last modified" date (e.g. "2022-06-05T20:21:37Z"). We'll need that later if we want to update the database with people's susbsequent changes to OpenStreetMap. Download it as follows:
```

Reply:
Why does this need modifying?

Reply:
I changed the download size to match the current size, and andygol changed the date to match too...

Reply:
We need to adjust this date in all parts of the documentation, not only here

Reply:
The download size says it is what the file currently is, but this is an example that will never be what it currently is except for one day.

Reply:
Rereading https://switch2osm.org/serving-tiles/manually-building-a-tile-server-20-04-lts/ , it does say at the top "These step-by-step instructions were written for [Ubuntu Linux](https://ubuntu.com/) 20.04 LTS (Focal Fossa), and were tested in May 2020" so I guess that the example download date of 2020 does make sense 

I've added a "currently" to the file size to hopefully make things clearer and left the dates, which tend to match when each guide was written.